sdk/core/core-lro/src/http/operation.ts

```diff
@@ -90,6 +90,7 @@ export function inferLroMode(inputs: {
   const pollingUrl = getOperationLocationPollingUrl({ operationLocation, azureAsyncOperation });
   const location = getLocationHeader(rawResponse);
   const normalizedRequestMethod = requestMethod?.toLocaleUpperCase();
+  logger.verbose(`LRO: inferLroMode: pollingUrl: ${pollingUrl}, location: ${location}, requestPath: ${requestPath}`);
```

Comment:
The configuration is already logged in L159. If you want to log the headers, that should already be done by the core client library when logging the response.

Reply:
I wanted to say.. the typical "operation-location" header is not found, moving on to the next case(in the if-else).
Would have been more helpful.
But I didn't know if that is indeed the typical case or if my message is misleading.

Let me know if you have suggestions along these lines. Or else I'll close this PR. 🥲

Reply:
> the typical "operation-location" header is not found

This could be determined from the core client logging, here is a snippet from the info logging for one of the LRO tests in @azure/ai-language-text:

> [node-tests] azure:ai-language-text:info Response status code: 202
[node-tests] azure:ai-language-text:info Headers: {
[node-tests]   "content-length": "0",
[node-tests]   "date": "Mon, 15 May 2023 22:58:59 GMT",
[node-tests]   "server": "Kestrel",
[node-tests]   "apim-request-id": "df48643c-3134-445a-a595-de8ac28c11a8",
[node-tests]   "operation-location": "https://endpointlanguage/analyze-text/jobs/fb0c9e7b-2d5b-4507-80ee-3d8e1e89060c?api-version=2023-04-01",
[node-tests]   "strict-transport-security": "max-age=31536000; includeSubDomains; preload",
[node-tests]   "x-content-type-options": "nosniff",
[node-tests]   "x-envoy-upstream-service-time": "310",
[node-tests]   "x-ms-client-request-id": "cd2de67f-09b0-427e-81e7-c30aa995a9c7",
[node-tests]   "x-ms-region": "East US"
[node-tests] }

Furthermore, core-lro logging prints the following:

> [node-tests] azure:core-lro:verbose LRO: Operation description: {
[node-tests]   metadata: { mode: 'OperationLocation' },
[node-tests]   operationLocation: 'https://endpointlanguage/analyze-text/jobs/fb0c9e7b-2d5b-4507-80ee-3d8e1e89060c?api-version=2023-04-01',
[node-tests]   resourceLocation: undefined
[node-tests] }

which clearly tells you where it polls from and where the processed resource, if any, will eventually be fetched from.

I am all for improving our logging but I also want to make sure it doesn't duplicate already logged information.

Reply:
ok